api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -540,6 +535,22 @@ public void getHandler_shouldHaveDefaultImageAndTextHandlersRegisteredBySpring()
 		Assert.assertNotNull(textHandler);
 	}
 	
+	@Test
+	@Verifies(value = "should purge complex obs when complex data file is missing from disk", method = "purgeObs(Obs)")
+	public void purgeComplexObs_shouldSucceedWhenCompleDataFileMissingFromDisk() throws Exception {
```

Comment:
@mks-d please for the next ones use this plugin
https://wiki.openmrs.org/display/docs/Generate+Test+Case+Plugin
you added the test using the old unit test style. thanks :)